kratos/python_interface/python_registry_utilities.py

```diff
@@ -17,6 +19,14 @@ def RegisterProcessesList(PythonModuleName, PythonRegistryListModule):
     __CheckRegistryListIsInModule(PythonRegistryListModule, "processes")
     __RegisterItemList(PythonModuleName, PythonRegistryListModule.python_processes_to_be_registered, "Processes")
 
+def RegisterStagesList(PythonModuleName, PythonRegistryListModule):
+    __CheckRegistryListIsInModule(PythonRegistryListModule, "stages")
+    __RegisterItemList(PythonModuleName, PythonRegistryListModule.python_stages_to_be_registered, "Stages")
+
+def RegisterMultistageOrchestratorsList(PythonModuleName, PythonRegistryListModule):
```

Comment:
We may rename it but eventually you might have orchestrators for different stuff (e.g. control problems or this sort of things). This is why I put the `Multistage` keyword in front.

About the single stage orchestrator, this is nothing but an instance of the `SequentialMultistageOrchestrator` with a single stage so I don't see the need of implementing it.

Reply:
This is what I want to avoid. Having the MultiStage in the front gives to the confusion that it is a type of stage and doesn't add any readability to it.

Reply:
OK. Fair enough. Removing it.